Assets/Scripts/PickUp.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PickUp : MonoBehaviour
+{
+    public Transform Hold;
```

Comment:
I think that we should save an indication of when an object can be picked up.
Whether using the selectable class to tell us weather this object is now being selected or something.
I think this will make it easier to use the same code for vr as well. As I'm not sure what does onMouseDown even do in vr.

Reply:
Isn`t onMouseDown the equivalent of the clenching your fist button in vr? Ill look into it and isn`t the colour change a good enough indication or do you want a hand to appear when you can pick something up?